chord_metadata_service/mohpackets/tests/test_models.py

```diff
@@ -0,0 +1,1483 @@
+        self.assertCountEqual(self.treatment.treatment_type, ["Endoscopic therapy", "Photodynamic therapy"])
+        self.assertEqual(self.treatment.is_primary_treatment, "Yes")
+        self.assertEqual(self.treatment.treatment_start_date, "2021-02")
+        self.assertEqual(self.treatment.treatment_end_date, "2022-09")
+        self.assertEqual(self.treatment.treatment_setting, "Neoadjuvant")
+        self.assertEqual(self.treatment.treatment_intent, "Palliative")
+        self.assertEqual(self.treatment.days_per_cycle, 1)
+        self.assertEqual(self.treatment.number_of_cycles, 3)
+        self.assertEqual(self.treatment.response_to_treatment_criteria_method,
+                         "Cheson CLL 2012 Oncology Response Criteria")
+        self.assertEqual(self.treatment.response_to_treatment, "Stable disease")
+
+    def test_unique_id(self):
+        with self.assertRaises(IntegrityError):
+            self.treatment = Treatment.objects.create(**self.valid_values)
+
+    def test_invalid_id(self):
+        invalid_values = get_invalid_ids()
+        for value in invalid_values:
+            self.valid_values["submitter_treatment_id"] = value
+            self.serializer = TreatmentSerializer(instance=self.treatment, data=self.valid_values)
+            self.assertFalse(self.serializer.is_valid())
+
+    def test_invalid_treatment_type(self):
+        invalid_values = get_invalid_choices()
+        for value in invalid_values:
+            with self.subTest(value=value):
+                self.valid_values["treatment_type"] = value
+            self.serializer = TreatmentSerializer(instance=self.treatment, data=self.valid_values)
+            self.assertFalse(self.serializer.is_valid())
+
+    def test_invalid_is_primary_treatment(self):
+        invalid_values = get_invalid_choices()
+        for value in invalid_values:
+            with self.subTest(value=value):
+                self.valid_values["is_primary_treatment"] = value
+            self.serializer = TreatmentSerializer(instance=self.treatment, data=self.valid_values)
+            self.assertFalse(self.serializer.is_valid())
+
+    def test_invalid_treatment_start_date(self):
+        invalid_values = get_invalid_dates()
+        for value in invalid_values:
+            with self.subTest(value=value):
+                self.valid_values["treatment_start_date"] = value
+            self.serializer = TreatmentSerializer(instance=self.treatment, data=self.valid_values)
+            self.assertFalse(self.serializer.is_valid())
+
+    def test_treatment_end_date(self):
+        invalid_values = get_invalid_dates()
+        for value in invalid_values:
+            with self.subTest(value=value):
+                self.valid_values["treatment_end_date"] = value
+            self.serializer = TreatmentSerializer(instance=self.treatment, data=self.valid_values)
+            self.assertFalse(self.serializer.is_valid())
```

Comment:
We should add the test to make sure end date is after start date. We can compare month and year at least